hack/verify-gofmt.sh

```diff
@@ -7,7 +7,7 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 bad_files=$(os::util::list_go_src_files | xargs gofmt -s -l)
 if [[ -n "${bad_files}" ]]; then
 	os::log::warning "!!! gofmt needs to be run on the following files: "
-	echo "${bad_files}"
+	echo "${bad_files}" >&2
```

Comment:
@liggitt wanted the UX of piping the output of this script into `xargs`, so this cannot go into `stderr`